### PR TITLE
[RFC] xbps-create: add -c/--changelog

### PR DESCRIPTION
**Pull request by ebfe**
Add option to set 'changelog' property. voidlinux/void-packages#3355

### CODE REVIEW

**Comment by xtraeme** [comment]
Please update NEWS and xbps-create.1.

Thanks

**Comment by ebfe** [comment]
Done

File: bin/xbps-create/xbps-create.1 [hunk 1]
```diff
@@ -87,6 +87,8 @@ If
 .Em symlink
 is a relative path, the symlink will be created relative to
 .Em target .
+.It Fl c, Fl -changelog Ar string
+The package changelog string.
```
Comment (xtraeme) [inline bin/xbps-create/xbps-create.1]: Please also bump the date in this manpage.

File: bin/xbps-create/main.c [hunk 2]
```diff
@@ -729,6 +731,8 @@ main(int argc, char **argv)
 		case 'C':
 			conflicts = optarg;
 			break;
+		case 'c':
+			changelog = optarg;
```
Comment (xtraeme) [inline bin/xbps-create/main.c]: missing `break`

File: bin/xbps-create/xbps-create.1 (hunk 1 above)
Comment (ebfe) [inline bin/xbps-create/xbps-create.1]: Done.

File: bin/xbps-create/main.c (hunk 2 above)
Comment (ebfe) [inline bin/xbps-create/main.c]: Argh, too much Go. Fixed.

**Pull request merged**